src/server/src/main/java/io/cassandrareaper/storage/cassandra/CassandraStorageFacade.java

```diff
@@ -81,30 +87,30 @@
 import systems.composable.dropwizard.cassandra.retry.RetryPolicyFactory;
 
 
-public final class CassandraStorage implements IStorage, IDistributedStorage {
-  private static final Logger LOG = LoggerFactory.getLogger(CassandraStorage.class);
+public final class CassandraStorageFacade implements IStorage, IDistributedStorage, IRepairRun {
+  private static final Logger LOG = LoggerFactory.getLogger(CassandraStorageFacade.class);
   private static final AtomicBoolean UNINITIALISED = new AtomicBoolean(true);
-  public final RepairSegmentDao repairSegmentDao;
+  public final CassRepairSegmentDao cassRepairSegmentDao;
   public final int defaultTimeout;
   final VersionNumber version;
   final UUID reaperInstanceId;
   private final com.datastax.driver.core.Cluster cassandra;
   private final Session session;
   private final ObjectMapper objectMapper = new ObjectMapper();
-  private final RepairRunDao repairRunDao;
+  private final CassRepairRunDao cassRepairRunDao;
   private PreparedStatement saveHeartbeatPrepStmt;
   private PreparedStatement deleteHeartbeatPrepStmt;
 
-  private final RepairUnitDao repairUnitDao;
-  private final RepairScheduleDao repairScheduleDao;
-  private final ClusterDao clusterDao;
-  private final EventsDao eventsDao;
-  private final MetricsDao metricsDao;
+  private final CassRepairUnitDao cassRepairUnitDao;
```

Comment:
I'm happy to make this change in the next PR if that's OK with you. It will take a bit of work and seems like it should be undertaken in a refactoring week so as not to distract from other work.